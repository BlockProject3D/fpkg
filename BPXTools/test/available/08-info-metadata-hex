export NAME="Info (Metadata + Hex)"
export COMMAND="-f test.bpx info -smx"
export DESC="Test the info command"
export STATUS=0