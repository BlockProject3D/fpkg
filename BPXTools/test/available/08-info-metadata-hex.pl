$Test = {
    Name => "Info (Metadata + Hex)",
    Command => "-f test/available/test.bpx info -smx",
    Description => "Test the info command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    return 1;
}
