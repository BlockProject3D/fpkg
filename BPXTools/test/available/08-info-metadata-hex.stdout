====> BPX Main Header <====
Type: P
Version: 1
File size: 3362131
Number of sections: 2
====> End <====

====> BPX TypeExt <====
04 04 50 4B 00 00 00 00 00 00 00 00 00 00 00 00 
====> End <====

====> BPX Section Header Table <====
Section #0:
	Type: 255
	Size (after compression): 7
	Size: 7
	Flags:  CheckWeak
Section #1:
	Type: 1
	Size (after compression): 3362036
	Size: 15318196
	Flags:  CompressXZ | CheckWeak
====> End <====

